python/orca/src/bigdl/orca/learn/tf2/pyspark_estimator.py

```diff
@@ -398,6 +399,9 @@ def predict(self,
         :param data_config: An optional dictionary that can be passed to data creator function.
         :param feature_cols: Feature column name(s) of data. Only used when data is a Spark
                DataFrame or an XShards of Pandas DataFrame. Default: None.
+        :param output_cols: Column name(s) of the model output data. Only used when data is
+               a Spark DataFrame, note the order of column name(s) should be consistent with the
+               model output data. Default: None.
```

Comment:
Need to add description of the default behavior. Maybe in a next PR after we finish this.

Reply:
sure.